arbitrum-docs/run-arbitrum-node/nitro/03-nitro-database-snapshots.md

```diff
@@ -0,0 +1,148 @@
+
+Finally, Nitro decompresses and extracts the snapshot archive, placing it in the database directory. Nitro will delete the archive after extracting it, so if you need to set up multiple nodes with the same snapshot, consider downloading it manually, as explained below.
+
+---
+
+## Downloading the snapshot from a URL
+
+Instead of letting Nitro search for the latest snapshot, you can provide a specific URL to download by setting the flag `--init.url` with the snapshot URL. If the URL points to a remote server, it should start with the `https://` protocol definition. Given the URL, Nitro will download the snapshot as described in the “Downloading the Latest Snapshot” section.
+
+Nitro also supports importing files from the local file system. In this case, you should provide the file path to `--init.url` starting with the prefix `file://` followed by the file path. Beware that when running Nitro inside a Docker container, you must mount a volume containing the provided snapshot using the docker flag `-v` (see [Docker documentation](https://docs.docker.com/reference/cli/docker/container/run/#volume)). Otherwise, the Nitro container running inside Docker won’t be able to find the snapshot in your local filesystem.
+
+---
+
+## Downloading the snapshot manually
+
+It is possible to download the snapshot manually and supply the archive instead of having Nitro download it.
+
+The first step is downloading the snapshot. The command below illustrates how to do that on the command line using wget. The `-c` flag tells the wget to continue the download from where it left off, which is helpful because snapshots can be huge files, and the download can fail mid-way. The `-P` flag tells wget to place the snapshot on the temporary dir.
+
+```shell
+wget -c -P /tmp "$SNAPSHOT_URL"
+```
+
+After downloading the snapshot, make sure to verify whether the checksum matches the one provided by the remote server. To fetch the checksum, you may run the command below. 
+
+```shell
+wget -q -O - "$SNAPSHOT_URL".sha256
+```
+
+Once you know the expected snapshot checksum, run the command below to compute the checksum of the downloaded snapshot. Then, compare both add see if they are the same. If they are not the same, consider redownloading the snapshot. You must provide a valid snapshot to Nitro; otherwise, it won’t work properly.
+
+```shell
+sha256sum $PATH_TO_SNAPSHOT
+```
+
+Finally, you can provide path to the downloaded snapshot archive to Nitro using the `--init.url` flag, as described in the “Download the Snapshot from a URL” section.
```

Comment:
```suggestion
It is possible to download the snapshot manually and supply the archive instead of having Nitro download it.

The first step is downloading the snapshot. The command below illustrates how to do that on the command line using wget. The `-c` flag tells the wget to continue the download from where it left off, which is helpful because snapshots can be huge files, and the download can fail mid-way. The `-P` flag tells wget to place the snapshot on the temporary dir.

```shell
wget -c -P /tmp "$SNAPSHOT_URL"
```

After downloading the snapshot, make sure to verify whether the checksum matches the one provided by the remote server. To fetch the checksum, you may run the command below. 

```shell
wget -q -O - "$SNAPSHOT_URL".sha256
```

Once you know the expected snapshot checksum, run the command below to compute the checksum of the downloaded snapshot. Then, compare both and see if they are the same. If they are not the same, consider redownloading the snapshot. You must provide a valid snapshot to Nitro; otherwise, it won’t work properly.

```shell
sha256sum $PATH_TO_SNAPSHOT
```

Finally, you can provide a path to the downloaded snapshot archive to Nitro using the `--init.url` flag, as described in the “Download the Snapshot from a URL” section.
```